src/prof.c

```diff
@@ -1750,11 +1817,13 @@ prof_dump_maps(bool propagate_err) {
 		goto label_return;
 	}
 
+#endif
```

Comment:
I don't think we want to move the `#endif` to here, e.g. the WIN32 case still needs the branch right above to set `ret = true;`